Tests/External/Plugins/ASCompletion.Tests/Completion/ASCompleteTests.cs

```diff
@@ -3,67 +3,19 @@
 using ASCompletion.Context;
 using ASCompletion.Model;
 using ASCompletion.Settings;
-using ASCompletion.TestUtils;
-using FlashDevelop;
+using ASCompletion.TestUtils.File;
 using HaXeContext;
 using NSubstitute;
 using NUnit.Framework;
-using PluginCore;
-using ScintillaNet;
-using ScintillaNet.Enums;
 
 //TODO: Sadly most of ASComplete is currently untestable in a proper way. Work on this branch solves it: https://github.com/Neverbirth/flashdevelop/tree/completionlist
 
 namespace ASCompletion.Completion
 {
     [TestFixture]
-    public class ASCompleteTests
+    public class ASCompleteTests : ScintillaTest
     {
-        private MainForm mainForm;
-        private ISettings settings;
-        private ITabbedDocument doc;
-
-        [TestFixtureSetUp]
-        public void FixtureSetUp()
-        {
-            mainForm = new MainForm();
-            settings = Substitute.For<ISettings>();
-            settings.UseTabs = true;
-            settings.IndentSize = 4;
-            settings.SmartIndentType = SmartIndent.CPP;
-            settings.TabIndents = true;
-            settings.TabWidth = 4;
-            doc = Substitute.For<ITabbedDocument>();
-            mainForm.Settings = settings;
-            mainForm.CurrentDocument = doc;
-            mainForm.StandaloneMode = false;
-            PluginBase.Initialize(mainForm);
-            FlashDevelop.Managers.ScintillaManager.LoadConfiguration();
-        }
-
-        [TestFixtureTearDown]
-        public void FixtureTearDown()
-        {
-            settings = null;
-            doc = null;
-            mainForm.Dispose();
-            mainForm = null;
-        }
-
-        private ScintillaControl GetBaseScintillaControl()
-        {
-            return new ScintillaControl
-            {
-                Encoding = System.Text.Encoding.UTF8,
-                CodePage = 65001,
-                Indent = settings.IndentSize,
-                Lexer = 3,
-                StyleBits = 7,
-                IsTabIndents = settings.TabIndents,
-                IsUseTabs = settings.UseTabs,
-                TabWidth = settings.TabWidth
-            };
-        }
+        private static TestFilePathInfo pathInfo = new TestFilePathInfo("ASComplete", "completion");
```

Comment:
What about calling that `testFile` instead of `pathInfo`? Would read better as `testFile.AS3("SimpleTest")`.